<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理员控制台</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="mainLayout">
    <item>
     <widget class="QWidget" name="sideBarWidget">
      <layout class="QVBoxLayout" name="sideBarLayout">
       <item>
        <widget class="QToolButton" name="userManagementBtn">
         <property name="text"><string>用户管理</string></property>
         <property name="toolButtonStyle"><enum>Qt::ToolButtonTextUnderIcon</enum></property>
         <property name="checkable"><bool>true</bool></property>
         <property name="autoExclusive"><bool>true</bool></property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="systemSettingsBtn">
         <property name="text"><string>系统设置</string></property>
         <property name="toolButtonStyle"><enum>Qt::ToolButtonTextUnderIcon</enum></property>
         <property name="checkable"><bool>true</bool></property>
         <property name="autoExclusive"><bool>true</bool></property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="deviceManagementBtn">
         <property name="text"><string>设备管理</string></property>
         <property name="toolButtonStyle"><enum>Qt::ToolButtonTextUnderIcon</enum></property>
         <property name="checkable"><bool>true</bool></property>
         <property name="autoExclusive"><bool>true</bool></property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="networkMonitorBtn">
         <property name="text"><string>网络监控</string></property>
         <property name="toolButtonStyle"><enum>Qt::ToolButtonTextUnderIcon</enum></property>
         <property name="checkable"><bool>true</bool></property>
         <property name="autoExclusive"><bool>true</bool></property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="alarmRuleManagementBtn">
         <property name="text"><string>告警规则</string></property>
         <property name="toolButtonStyle"><enum>Qt::ToolButtonTextUnderIcon</enum></property>
         <property name="checkable"><bool>true</bool></property>
         <property name="autoExclusive"><bool>true</bool></property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="alarmDisplayBtn">
         <property name="text"><string>告警展示</string></property>
         <property name="toolButtonStyle"><enum>Qt::ToolButtonTextUnderIcon</enum></property>
         <property name="checkable"><bool>true</bool></property>
         <property name="autoExclusive"><bool>true</bool></property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="dataAnalysisBtn">
         <property name="text"><string>数据分析</string></property>
         <property name="toolButtonStyle"><enum>Qt::ToolButtonTextUnderIcon</enum></property>
         <property name="checkable"><bool>true</bool></property>
         <property name="autoExclusive"><bool>true</bool></property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation"><enum>Qt::Vertical</enum></property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="logoutBtn">
         <property name="text"><string>登出</string></property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitBtn">
         <property name="text"><string>退出</string></property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="mainStackedWidget">
      <property name="minimumWidth"><number>800</number></property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
